config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,53 @@
+Rack::Attack.enabled = ENV['ENABLE_RACK_ATTACK'] || Rails.env.production?
+
+module Rack
+  # Configure rack-attack gem
+  class Attack
+    ### Configure Cache ###
+
+    # If you don't want to use Rails.cache (Rack::Attack's default), then
+    # configure it here.
+    #
+    # Note: The store is only used for throttling (not blocklisting and
+    # safelisting). It must implement .increment and .write like
+    # ActiveSupport::Cache::Store
+
+    # Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
+
+    ### Blacklisted IPs ###
+
+    bad_ips = ENV['BLOCKED_IPS'] ? ENV['BLOCKED_IPS'].split(',') : []
```

Comment:
env var has been added to prod environment: https://dashboard.heroku.com/apps/dahlia-production/settings